src/server/index.spec.tsx

```diff
@@ -106,5 +108,102 @@ describe(__filename, () => {
         expect(response.text).toContain(expectedHTML);
       });
     });
+
+    describe('NODE_ENV=development', () => {
+      let proxy: express.Application;
+      let proxyServer: http.Server;
+
+      const devEnv = {
+        NODE_ENV: 'development',
+        REACT_APP_CRA_PORT: 60123,
+      };
+
+      beforeEach(() => {
+        proxy = express();
+        proxyServer = proxy.listen(devEnv.REACT_APP_CRA_PORT);
+      });
+
+      afterEach(() => {
+        proxyServer.close();
+      });
+
+      it('creates an Express server that uses a proxy', async () => {
+        const proxyContent = 'Hello, I am a proxy';
```

Comment:
This is more like `proxiedContent`, right? Can you make it more realistic? Perhaps: `Hello, I am the create-react-app server` ... or something.

Reply:
correct, thanks